test/fixtures/document_with_special_heading.md

```diff
@@ -0,0 +1,4 @@
+# Sample `markdown` with _special_ **heading**
+
+Sometimes heading contains more than just one text child, the current file is
+there to test just that. 
```

Comment:
This prose text is unnecessary.

```suggestion
```

Reply:
It's to explain the purpose of the file, I'd prefer to keep it as is.